app/assets/javascripts/qz/ticket-popup.js

```diff
@@ -0,0 +1,85 @@
+var privateKey = "-----BEGIN RSA PRIVATE KEY-----\n" +
```

Comment:
Ok, but the only benefit to not using an ajax call to retrieve a signed message from an endpoint on the server is your last point, which doesn't really seem overly onerous to me. It's a one-time **optional** configuration. Hardly a difficult hurdle for someone seriously considering deploying the ofn...

In fact, by *not* baking the secret key into the code, we allow greater flexibility: anyone who wants to deploy an instance of the ofn can always compile their own client and use their own secret key.